strings/names/mime.txt

```diff
@@ -1,24 +1,50 @@
+Ceci N'est Pas Une Mime
+Cousteau
+Eastern Standard Mime
+Everywhere at the End of Mime
+Ford Thirtythree
+Harpo Marx
+Hushed Henri
+Instrumental
 Invisible Man
 Lemon Mime
+Loudmouth
 Marcel
 Marcel Mime
 Mime
+Mime is Relative
 Mr Beret
 Mr Mime
 Mr Mute
 Mute
+Nickel and Mime
+The Noiseless Wonder
 Omerta
+One Hand Clapping
+Oreo
 Oui Oui
 Pantomime
+Parle Francais
```

Comment:
if you want to be grammatically correct here its `Parles` or `Parlez`

Reply:
Added, thanks!